Library/Homebrew/rubocops/formula_desc_cop.rb

```diff
@@ -78,8 +79,13 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
           end
 
           # Check if formula's desc starts with formula's name
-          return unless regex_match_group(desc, /^#{@formula_name} /i)
-          problem "Description shouldn't start with the formula name"
+          if regex_match_group(desc, /^#{@formula_name} /i)
+            problem "Description shouldn't start with the formula name"
+          end
+
+          # Check if a full stop is used at the end of a formula's desc
+          return unless regex_match_group(desc, /\.$/)
+          problem "Description shouldn't end with a full stop"
```

Comment:
Could you add an `autocorrect` for this on line 105 below? Something like this would work: `correction.gsub!(/\.$/, "")`